src/govuk/components/accordion/accordion.yaml

```diff
@@ -18,7 +18,7 @@ params:
 - name: items
   type: array
   required: true
-  description: An array of sections within the accordion.
+  description: An array of sections within the accordion, undefined values are skipped.
```

Comment:
>  I wasn't sure how best to describe this in the docs

We may not need to update these descriptions, if we think that this is an expected behaviour from our users, which I'd argue it is.

Reply:
Yeah, that is why I opened it as WIP - it seems like it needs to be documented, but wasn't sure the best way.

This is certain a way!

Reply:
Just reflecting on this, with some arrays now working like this, I feel like it should be explicit?

Happy to take the review comment and change, else I can this can be resolved?